lib/build_targets/package.dart

```diff
@@ -102,16 +102,19 @@ class DotnetTpk extends TizenPackage {
         getEngineArtifactsDirectory(buildInfo.targetArch, buildMode);
     final Directory embedderDir =
         getEmbedderArtifactsDirectory(apiVersion, buildInfo.targetArch);
-
     final File engineBinary = engineDir.childFile('libflutter_engine.so');
     final File embedder = embedderDir.childFile('libflutter_tizen_$profile.so');
     final File icuData = engineDir.childFile('icudtl.dat');
+    final File pluginsExtraInfo =
+        project.directory.childFile('.flutter-tizen-plugins-extra-info');
```

Comment:
I think it would be better if the .flutter-tizen-plugins-extra-info file is located in the app root rather than flutter/ephemeral. (In the case of C#, Runner.deps.json is located in the app root)
and I think `.json` extension could kept.
so the file name could be app.deps.json, flutter-tizen.deps.json or something better. If you have a better idea, please let me know.